packages/mui-system/src/colorManipulator/index.ts

```diff
@@ -0,0 +1,2 @@
+'use client'
+export * from './colorManipulator'
```

Comment:
We are missing semicolons in these files, how did the CI pass? 😕 I created https://github.com/mui/material-ui/pull/41636 to address this.